google/cloud/security/scanner/scanner.py

```diff
@@ -240,15 +224,43 @@ def _write_violations_output(logger, violations):
                 'member': '{}:{}'.format(member.type, member.name)
             }
 
-def _upload_csv_to_gcs(logger, output_path, output_filename, csv_name):
+def _output_results(logger, all_violations, **kwargs):
+    """Send the output results.
+
+    Args:
+        logger: The logger.
```

Comment:
Do you need to pass the logging instance? Shouldn't it already be available to the method?